src/mpi/romio/adio/ad_gpfs/bg/ad_bg_pset.c

```diff
@@ -16,14 +16,14 @@
 // #define bridgeringaggtrace 1
 
 #include <stdlib.h>
+#include <stdbool.h>
```

Comment:
Can we drop the usage of `stdbool.h` in favor of `MPL_bool`? Our bool usage throughout the code is inconsistent, but if you use what is provided by MPL, we can fix it later and get it right for the rest of the repo (hopefully).